### ISSUE TITLE
Fusions being prioritized over regular transactions

### ISSUE THREAD

**Issue by zpalmtree** [turtlecoin/turtlecoin#767]
As the title says, fusions (zero fee) are being included over transactions with a fee. The relevant chunk of code is here:

https://github.com/turtlecoin/turtlecoin/blob/development/src/CryptoNoteCore/Core.cpp#L2499

It could also do with being cleaned up a bit, the logic is a little hard to follow.

**Comment by brandonlehmann** [comment]
It would be awesome if while in there that part of #749 is handled as part of it
